watchtower/conf.go

```diff
@@ -1,14 +1,63 @@
-// +build !experimental
-
 package watchtower
 
-// Conf specifies the watchtower options that be configured from the command
-// line or configuration file. In non-experimental builds, we disallow such
-// configuration.
-type Conf struct{}
+import (
+	"time"
+
+	"github.com/lightningnetwork/lnd/lncfg"
+)
+
+// Conf specifies the watchtower options that can be configured from the command
+// line or configuration file.
+type Conf struct {
+	RawListeners []string `long:"listen" description:"Add interfaces/ports to listen for peer connections"`
+
+	ReadTimeout time.Duration `long:"readtimeout" description:"Duration the watchtower server will wait for messages to be received before hanging up on clients"`
+
+	WriteTimeout time.Duration `long:"writetimeout" description:"Duration the watchtower server will wait for messages to be written before hanging up on client connections"`
+}
 
-// Apply returns an error signaling that the Conf could not be applied in
-// non-experimental builds.
+// Apply completes the passed Config struct by applying any parsed Conf options.
```

Comment:
Looks to be over the column limit at a glance?

Reply:
exactly 80 :)